### DISCUSSION TITLE
feature request

### DISCUSSION THREAD

**Discussion by Slydylan** [MrHydralisk/Lethal-Company-Enhanced-Radar-Booster#1] [Ideas]
can you make it so players can teleport themselves to the location of a radar booster by typing like.. TP Sly - Greg or w/e?
would be a great solos tool

**Comment by MrHydralisk** [comment]
Answered in discord already, but just in case: will see what can be done

**Comment by Slydylan** [comment]
> Answered in discord already, but just in case: will see what can be done
discord? didnt know you were on discord. either way if this becomes a feature ill be excited

**Reply by MrHydralisk** [reply]
Oh, I guess then someone else noticed your request before I did and write it on Discord. Not really keeping my own Discord server for modding, but had this mod referenced on Lethal Company Modding server. So to update you on that, I did had quite the progress on changing inverse teleporter target point few min ago, so it will likely become feature.

**Comment by MrHydralisk** [comment]
Added Feature: 
- When Radar Booster selected on Ship Monitor and Inverse Teleporter activated Player will be teleported to selected Radar Booster.

**Comment by Slydylan** [comment]
ah sweet. Cheers for that!!
